assets/app.js

```diff
@@ -859,67 +859,79 @@ new Vue({
             console.log(this.settings)
         },
         exportProduct: function (product) {
-            dict_materials = this.materials
-            dict_metatags = this.metatags
-
-            // localize materials
-            localized_materials = {}
-            localized_metatags = {}
-            export_languages = []
-            this.settings.supportedLanguages.forEach(function (language) {
-                if (language.status) {
-                    export_languages.push(language.id)
-                    localized_materials[language.id] = []
-                    localized_metatags[language.id] = []
-
-                    for (var i = 0; i < product.materials.length; i++) {
-                        localized_materials[language.id].push(product.materials[i][language.id])
+            msg = "";
+            if (!product.base_product && !product.materials.length) {
+                msg = "Are you sure to save empty `baseproduct` and empty `materials` in `" + product.modelCode +"` ?"
+            } else if (!product.materials.length) {
+                msg = "Are you sure to save empty `material` in `" + product.modelCode + "` ?"
+            } else if(!product.base_product) {
+                msg = "Are you sure to save empty `baseproduct` in `" + product.modelCode + "` ?"
+            }
 
-                    }
-                    for (var i = 0; i < product.metatags.length; i++) {
-                        //localized_metatags[language.id].push(product.metatags[i][language.id])
+            proceed = true
```

Comment:
ha? please remove.

Reply:
@Solomonic 
I use ` proceed = true` for case that don't have any empty base_product and material.(normal case)